apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -0,0 +1,246 @@
+            ? html`<p>
+                It doesn't look like you have any PrairieLearn courses.
+                <a href="/pl/request_course" class="btn btn-success">Go request a course</a>
+              </p>`
+            : html`
+                <p>Your courses:</p>
+                <ul>
+                  ${courses.map((course) => {
+                    return html` <li>
+                      <a href="/pl/course/${course.id}">${course.short_name}: ${course.title}</a>
+                    </li>`;
+                  })}
+                </ul>
+                <form method="POST">
+                  <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+                  <label>Connect ${courseName} with:
+                  <div class="input-group">
+                    <select class="custom-select" id="onepicker" name="unsafe_course_instance_id">
+                      <option value="" disabled selected>
+                        Select an existing course instance...
+                      </option>
+                      ${courses.map((course) => {
+                        const course_cis = course_instances.filter(
+                          (ci) => ci.course_id === course.id,
+                        );
+                        return html`
+                          <optgroup label="${course.short_name}: ${course.title}">
+                            ${course_cis.map((ci) => {
+                              return html`<option value="${ci.id}">
+                                ${course.short_name} -- ${ci.long_name} (${ci.short_name})
+                              </option>`;
+                            })}
+                          </optgroup>
+                        `;
+                      })}
+                    </select>
+                    </label>
+                  </div>
+                  <button class="btn btn-primary" id="saveButton" disabled>Save</button>
+                </form>
+              `}
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+export function Lti13CourseNavigationNotReady({
+  courseName,
+  resLocals,
+}: {
+  courseName: string;
+  resLocals: Record<string, any>;
+}): string {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'lti13_course_naviation',
+          pageTitle: 'Course',
+        })}
+      </head>
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'lti13_course_navigation',
+        })}
+        <main class="container mb-4">
+          <h1 class="h1">Welcome to PrairieLearn</h1>
+          <h2 class="h2">... but your course isn't ready yet!</h2>
+
+          <p>An instructor has not yet configured ${courseName} in PrairieLearn.</p>
+          <p>Please come back later.</p>
```

Comment:
Let's add a "home" link here so that students can navigate to other courses after authing, even if this particular course isn't linked to an instance yet.

Reply:
Added in 24e76c7.